File: app/src/main/java/com/oidiotlin/classmanager/utils/database/Person.java
package com.oidiotlin.classmanager.utils.database;

/**
 * Created by OIdiot on 2017/1/24.
 * Project: ClassManager
 */

public class Person {
    private String studentNumber;   // 学号
    private String name;    // 姓名
    private String pinyin;  // 拼音
    private String gender;  // 性别
    private String nativeProvince;  // 籍贯
    private String dormitory;   // 寝室
    private String birthday;    // 生日
    private String phoneNumber; // 手机
    private String position;    // 职务
    private int participation;  // 活动参与度

    public String getPinyin() {
        return pinyin;
    }

    public void setPinyin(String pinyin) {
        this.pinyin = pinyin;
    }

    public String getStudentNumber() {
        return studentNumber;
    }

    public void setStudentNumber(String studentNumber) {
        this.studentNumber = studentNumber;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getNativeProvince() {
        return nativeProvince;
    }

    public void setNativeProvince(String nativeProvince) {
        this.nativeProvince = nativeProvince;
    }

    public String getDormitory() {
        return dormitory;
    }

    public void setDormitory(String dormitory) {
        this.dormitory = dormitory;
    }

    public String getBirthday() {
        return birthday;
    }

    public void setBirthday(String birthday) {
        this.birthday = birthday;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public int getParticipation() {
        return participation;
    }

    public void setParticipation(int participation) {
        this.participation = participation;
    }
}

File: server/applications/Person.php
<?php

class Person {

	private $id;
	private $studentNumber;
	private $pinyin;
	private $gender;
	private $nativeProvince;
	private $dormitory;
	private $birthday;
	private $phoneNumber;
	private $position;
	private $participation;

	public function getId() {
	    return $this->id;
	}
	
	public function setId($id) {
	    $this->id = $id;
	    return $this;
	}
	
	public function getStudentNumber() {
	    return $this->studentNumber;
	}
	
	public function setStudentNumber($studentNumber) {
	    $this->studentNumber = $studentNumber;
	    return $this;
	}
	
	public function getPinyin() {
	    return $this->pinyin;
	}
	
	public function setPinyin($pinyin) {
	    $this->pinyin = $pinyin;
	    return $this;
	}
	
	public function getGender() {
	    return $this->gender;
	}
	
	public function setGender($gender) {
	    $this->gender = $gender;
	    return $this;
	}
	
	public function getNativeProvince() {
	    return $this->nativeProvince;
	}
	
	public function setNativeProvince($nativeProvince) {
	    $this->nativeProvince = $nativeProvince;
	    return $this;
	}
	
	public function getDormitory() {
	    return $this->dormitory;
	}
	
	public function setDormitory($dormitory) {
	    $this->dormitory = $dormitory;
	    return $this;
	}
	
	public function getBirthday() {
	    return $this->birthday;
	}
	
	public function setBirthday($birthday) {
	    $this->birthday = $birthday;
	    return $this;
	}
	
	public function getPhoneNumber() {
	    return $this->phoneNumber;
	}
	
	public function setPhoneNumber($phoneNumber) {
	    $this->phoneNumber = $phoneNumber;
	    return $this;
	}
	
	public function getPosition() {
	    return $this->position;
	}
	
	public function setPosition($position) {
	    $this->position = $position;
	    return $this;
	}
	
	public function getParticipation() {
	    return $this->participation;
	}
	
	public function setParticipation($participation) {
	    $this->participation = $participation;
	    return $this;
	}
	
}

?>